ehrapy/plot/_survival_analysis.py

```diff
@@ -293,5 +296,132 @@ def kaplan_meier(
 
     if not show:
         return ax
+
     else:
         return None
+
+
+def coxph_forestplot(
+    coxph: CoxPHFitter,
+    labels: list[str] | None = None,
+    fig_size: tuple = (10, 10),
+    t_adjuster: float = 0.1,
+    ecolor: str = "dimgray",
+    size: int = 3,
+    marker: str = "o",
+    decimal: int = 2,
+    text_size: int = 12,
+    color: str = "k",
+):
+    """Plots a forest plot of the Cox Proportional Hazard model.
+    Inspired by `zepid.graphics.EffectMeasurePlot <https://readthedocs.org>`_ (zEpid Package, https://pypi.org/project/zepid/).
+
+    Args:
+        coxph: Fitted CoxPHFitter object.
+        labels: List of labels for each coefficient, default uses the index of the coxph.summary.
+        fig_size: Width, height in inches.
+        t_adjuster: Adjust the table to the right.
+        ecolor: Color of the error bars.
+        size: Size of the markers.
+        marker: Marker style.
+        decimal: Number of decimal places to display.
+        text_size: Font size of the text.
+        color: Color of the markers.
+
+    Examples:
+        >>> import ehrapy as ep
+        >>> adata = ep.dt.mimic_2(encoded=False)
+        >>> adata_subset = adata[:, ["mort_day_censored", "censor_flg", "gender_num", "afib_flg", "day_icu_intime_num"]]
+        >>> coxph = ep.tl.coxph(adata_subset, event_col="censor_flg", duration_col="mort_day_censored")
+        >>> ep.pl.coxph_forestplot(coxph)
+
+        .. image:: /_static/docstring_previews/coxph_forestplot.png
+
+    """
+
+    data = coxph.summary
```

Comment:
`data` is not a very precise variable name ^_^ But it might be okay here

Reply:
see earlier comment, maybe the plot can use what is stored in `adata.var`, and ignore the coxph object